src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1432,9 +1442,54 @@
 								marqueeWidget.reset();
 							}
 						}
+
 					}
 				}
-			};
+			}
+
+			prototype._enableSwipe = function (element) {
+				ns.event.enableGesture(
+					element,
+
+					new gesture.Drag({
+						threshold: 10,
+						blockVertical: true
+					})
+				);
+
+				element.addEventListener("dragstart", this._itemDragStart);
+				element.addEventListener("drag", this._itemDragging);
+			}
+
+			// TODO: this should be called on selection change.
+			prototype._disableSwipe = function (element) {
+				element.removeEventListener("dragstart", this._itemDragStart);
+				element.removeEventListener("drag", this._itemDragging);
+			}
+
+			prototype._itemDragStart = function () {
+				lastDragDeltaX = 0;
+			}
+
+			prototype._itemDragging = function (event) {
+				var target = event.target,
+					gesture = event.detail,
+					swipeIcon = target.querySelector(selectors.SWIPE_ICON),
+					currentClipPath,
+					newClipPath;
+
+				if (swipeIcon) {
+					currentClipPath = getComputedStyle(swipeIcon).clipPath.match(/[0-9]+%/);
```

Comment:
You can define local variable like `PERCENT_REGEXP = /[0-9]+%/`

Reply:
Added.